widget-reference/button.md

```diff
@@ -12,10 +12,16 @@ description: >-
 | **Button Style** | Sets the style of the button. You can choose from three button styles: Primary, Secondary & Danger Button. |
 | **Visible** | Controls widget's visibility on the page. When turned off, the widget will not be visible when the app is published  |
```

Comment:
@JoshHeng the gif doesn't show the user using re-captcha and sending the token to the server. Can we make this a separate Page with clear instructions to create a recaptcha site key, configure the button and send the token to the backend. Also need information on how the server will verify the token

Reply:
I've separated the integration instructions to a separate page. 
I don't think the button gif should show adding the recaptcha key/verification as I think it should just show the client side/appsmith configuration of the button

Reply:
@JoshHeng I think there was some confusion here. I mean, can we have a separate gif not change the button gif. Which shows a user adding the key to the button, creating an API in the API pane and binding the buttons token to the API body i,e {{ Button1.token }}